### PR TITLE
Gemfile command

### PR DESCRIPTION
**Pull request by agwozdowski**
Resolved issue #35

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - #18: Add bundler-audit which checks for vulnerable versions of gems
 - #21: Replace readme validator with required files validators, which allows us to
   define what files you want to require in your project
+- #35: Added gemfile validator which check if Gemfile contains gems from local path
```
Comment (mensfeld) [inline CHANGELOG.md]: checks

**Comment by agwozdowski** [comment]
@mensfeld Typo corrected. Run gemfile validator if Gemfile exists

**Comment by nijikon** [comment]
@agwozdowski please squash to one commit and rebase on top of master

**Comment by nijikon** [comment]
Also, please append ```Fixes #35``` at the end of the commit message. Commit message should look like this. ```Gemfile local path validator, Fixes #35```

**Comment by agwozdowski** [comment]
@nijikon I've just squashed all commit to one and rebase on top of master

**Pull request merged**